app/models/claim.rb

```diff
@@ -161,9 +161,9 @@ class Claim < ApplicationRecord
 
   scope :unsubmitted, -> { where(submitted_at: nil) }
   scope :submitted, -> { where.not(submitted_at: nil) }
-  scope :awaiting_decision, -> { submitted.left_outer_joins(:decision).where(decisions: {claim_id: nil}) }
-  scope :approved, -> { joins(:decision).where("decisions.result" => :approved) }
-  scope :rejected, -> { joins(:decision).where("decisions.result" => :rejected) }
+  scope :awaiting_decision, -> { submitted.left_outer_joins(:decisions).where(decisions: {claim_id: nil}) }
+  scope :approved, -> { joins(:decisions).where("decisions.result" => :approved) }
+  scope :rejected, -> { joins(:decisions).where("decisions.result" => :rejected) }
```

Comment:
With these two scopes, it's likely that claims that have been approved then subsequently rejected will show in both scopes. Is this something that will come later?

Reply:
Yes - the next PR in the chain introduces some far more robust testing around these when it introduces the idea of something being able to move back to undecided.